notebooks/curation_api/README.md

```diff
@@ -0,0 +1,25 @@
+# Curation API notebooks
```

Comment:
change to discovery and say formerly know as curation notebooks through out the doc